bin/package

```diff
@@ -102,14 +105,15 @@ echo "Testing produced binary..."
 echo "Zipping binary..."
 cd dist
 
-ZIP_NAME="gdc-client_${VERSION}_${TARGET_ENVIRONMENT}_x64.zip"
+ZIP_NAME="gdc-client_${VERSION}_${TARGET_ENVIRONMENT}_x64"
```

Comment:
I'm not sure I understand this change. Can we keep the `.zip` extension in the `ZIP_NAME` variable and make a variable before this without the extension? Then the pre-name variable can be used in the rest of the action? 
```suggestion
PRE_ZIP_NAME="gdc-client_${VERSION}_${TARGET_ENVIRONMENT}_x64"
ZIP_NAME="$PRE_ZIP_NAME.zip"
```

We could also use bash built-in functionality to remove the `.zip` when we need it at the end.

```bash
$ ZIP_NAME="gdc-client-windows.zip"
$ echo ${ZIP_NAME%.zip}
gdc-client-windows
```

Reply:
This change is mainly to power this command https://github.com/NCI-GDC/gdc-client/blob/9e1957b56493a3975721f625e1abefca59bef512/.github/workflows/ci.yaml#L43
which sets the artifact name. I agree, your implementation above is neater